lang/cs/Org.Apache.REEF.Wake/Time/Runtime/RuntimeClock.cs

```diff
@@ -57,15 +64,30 @@ public sealed class RuntimeClock : IClock
         /// <param name="runtimeStartHandler">The runtime start handler</param>
         /// <param name="runtimeStopHandler">The runtime stop handler</param>
         /// <param name="idleHandler">The idle handler</param>
+        /// <param name="oldStartHandler">Start handlers prior to REEF-1372</param>
+        /// <param name="oldStopHandler">Stop handlers prior to REEF-1372</param>
+        /// <param name="oldRuntimeStartHandler">Runtime start handlers prior to REEF-1372</param>
+        /// <param name="oldRuntimeStopHandler">Runtime stop handlers prior to REEF-1372</param>
+        /// <param name="oldIdleHandler">Idle handlers prior to REEF-1372</param>
+#pragma warning disable 618
```

Comment:
Yes, it disables the warnings about the `Obsolete` named parameters below. Given that this implements support for `Obsolete` APIs, I don't think it itself should be considered `Obsolete`.